packages/client/CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Add config option `lighterGapFill` (true by default) to control gap filling strategy
```

Comment:
Would it make sense if we'd configure this by enum: `gapFillStrategy: 'full'` or `gapFillStrategy: 'light'`? It would be easier to communicate if the strategies have exact names. 

The boolean logic is quite easy, but not as straightforward as it could be: i.e. if we want full gap fill, we need to set lighter to `false`.